packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -61,12 +61,12 @@ export function createApp() {
     .get<SingleEntityRequest>('entity/:projectCode/:entityCode', handleWith(SingleEntityRoute))
     .get<EntitiesRequest>('entities', handleWith(EntitiesRoute))
     .get<SurveysRequest>('surveys', handleWith(SurveysRoute))
-    .get<SurveyResponsesRequest>('surveyResponses', handleWith(SurveyResponsesRoute))
+    .get<SubmissionHistoryRequest>('submissionHistory', handleWith(SubmissionHistoryRoute))
     .get<SurveyRequest>('surveys/:surveyCode', handleWith(SurveyRoute))
     .get<ProjectsRequest>('projects', handleWith(ProjectsRoute))
     .get<LeaderboardRequest>('leaderboard', handleWith(LeaderboardRoute))
     .get<ProjectRequest>('project/:projectCode', handleWith(ProjectRoute))
-    .get<RecentSurveysRequest>('recentSurveys', handleWith(RecentSurveysRoute))
+    .get<TopSurveysRequest>('topSurveys', handleWith(TopSurveysRoute))
```

Comment:
Same here. I actually think it would be a pretty good option to name this simply SurveysRoute since the api doesn't need to know that it's for the recent section and it's pretty expected for the latest records to be returned anyway.